src/vs/workbench/contrib/chat/common/chatModel.ts

```diff
@@ -138,16 +141,16 @@ export class Response implements IResponse {
 			this._updateRepr(quiet);
 		} else if ('placeholder' in responsePart) {
 			// Add a new resolving part
-			const responsePosition = this._responseParts.push({ string: new MarkdownString(responsePart.placeholder), resolving: true }) - 1;
+			const responsePosition = this._responseParts.push({ string: new MarkdownString(responsePart.placeholder), isPlaceholder: true }) - 1;
 			this._updateRepr(quiet);
 
 			responsePart.resolvedContent?.then((content) => {
 				// Replace the resolving part's content with the resolved response
 				if (typeof content === 'string') {
-					this._responseParts[responsePosition] = { string: new MarkdownString(content), resolving: true };
+					this._responseParts[responsePosition] = { string: new MarkdownString(content), isPlaceholder: true };
```

Comment:
I guess I don't follow why we use MarkdownStrings when it seems like the renderer always instantiates new MarkdownStrings anyway

Reply:
The `ChatModel` has always dealt with `MarkdownString`s AFAIK https://github.com/microsoft/vscode/commit/d3adf9ce12343bd6c4b192689d260b13d644315d#diff-5919fee4d8fd3a0064e90fc1fa2931424b8c0cf04f5359b784f96c9fb4cde2ffL192-R231 but I can clean that up separately since it would simplify things

Reply:
Ah I guess so- probably I was expecting that the options on MarkdownString would be used at some point and I would want to manage those in the model, but I don't think that has happened- it's fine, you don't have to change it